rust/worker/src/system/executor.rs

```diff
@@ -69,14 +70,24 @@ where
                 message = channel.recv() => {
                     match message {
                         Some(mut message) => {
-                            message.handle(&mut self.handler,
-                                &ComponentContext{
+                            let parent_span: tracing::Span;
```

Comment:
Would the code run if we turn off tracing?

Reply:
If not, does it add overhead even the tracing is not needed?

Reply:
By turning off tracing, do you mean setting a level less than `TRACE`?